plugins/inputs/logfile/logfile_test.go

```diff
@@ -400,7 +402,12 @@ func TestLogsFileAutoRemoval(t *testing.T) {
 			os.Remove(tmpfile2.Name())
 		}
 	}()
+
+	wg.Add(1)
+	
 	go func() {
+		defer wg.Done()
```

Comment:
If we have this, do we still need the sleep?

Reply:
Not anymore. I have tested without the sleep.

Reply:
Can you remove the sleep on line [411](https://github.com/aws/amazon-cloudwatch-agent/pull/406/files#diff-c757df81e1ad75b84c4059eef343eb2b855921ec0206e402b73f3e030c592b48R411) then?

Reply:
Deleted. Have not noticed that. Only notice the sleep I intend to update though. Thanks for reminding.